components/Layout/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import Link from "next/link";
+import Wrapper from "../Wrapper";
+import LinkItem from "../ui/LinkItem";
+
+const Header = () => {
+  return (
+    <Wrapper>
+      <div className="flex flex-col justify-between gap-5 py-12 text-xl lg:flex-row sm:text-2xl -tracking-tighter sm:tracking-normal">
+        <div>Ape Unit</div>
+        <div className="max-w-189.25 xl:max-w-201.5">
+          Unit➇ is a pioneering technology company specialing in decentralised
+          technologies that creates end-to-end digital experiences for protocols
+          including{" "}
+          <span className="text-purple">
```

Comment:
Space before and after should be used if it is the only choice you have. 
In this case, you can use `px-1` as a padding
```suggestion
          <span className="text-purple px-1">
```